yt/frontends/cf_radial/io.py

```diff
@@ -0,0 +1,67 @@
+"""
+CF-Radial-specific IO functions
+
+
+
+"""
+
+import numpy as np
+
+from yt.utilities.io_handler import BaseIOHandler
+
+
+class CFRadialIOHandler(BaseIOHandler):
+    _particle_reader = False
+    _dataset_type = "cf_radial"
+
+    def _read_particle_coords(self, chunks, ptf):
+        # This needs to *yield* a series of tuples of (ptype, (x, y, z)).
+        # chunks is a list of chunks, and ptf is a dict where the keys are
+        # ptypes and the values are lists of fields.
+        pass
+
+    def _read_particle_fields(self, chunks, ptf, selector):
+        # This gets called after the arrays have been allocated.  It needs to
+        # yield ((ptype, field), data) where data is the masked results of
+        # reading ptype, field and applying the selector to the data read in.
+        # Selector objects have a .select_points(x,y,z) that returns a mask, so
+        # you need to do your masking here.
+        pass
```

Comment:
I think you can just remove those altogether

Reply:
removed